<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointForm</class>
 <widget class="QWidget" name="PointForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="5">
    <widget class="QLabel" name="label_p2_1">
     <property name="text">
      <string> p2(</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_p1_3">
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_p1_2">
     <property name="maximumSize">
      <size>
       <width>4</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>,</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_y1">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_x1">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLineEdit" name="lineEdit_y2">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="lineEdit_x2">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_p2_2">
     <property name="maximumSize">
      <size>
       <width>4</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="label_p2_3">
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_p1_1">
     <property name="text">
      <string>p1(</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
